api/test/unit/views/test_health_views.py

```diff
@@ -33,9 +33,9 @@ def test_health_check_calls__check_db(api_client):
 
 def test_health_check_es_timed_out(api_client):
     mock_health_response(timed_out=True)
-    pook.on()
```

Comment:
This and the other changes in this test module are cleanup. Using `pook.on` and `pook.off` like this means that if `api_client.get` results in an exception (which is a test failure), then pook would never get turned off. It's exactly what a context manager is for anyway.